content/blog/2021-02-12-february-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,172 @@
+the team as a Technical Product Manager. We are thrilled he's here!
+
+[**Batuhan Taskaya**](https://www.linkedin.com/in/batuhan-osman-taskaya-7803b61a0/)
+joins us as a DVC Software Engineer working on the Python core. Batuhan is
+excited to work on open source full time and we are excited to have him do so!
+
+[**Jeny De Figueiredo**](https://www.linkedin.com/in/jenifer-de-figueiredo/) is
+involved in the Seattle area data science community at Data Circles and is a
+WiDS Puget Sound Ambassador. She joins us as our new Community Manager and is
+looking forward to further building and engaging the community in MLOps! (Hi!
+This is me. 🙋🏻‍♀️ I'll be writing Heartbeat!)
+
+[**Roger Parent**](https://www.linkedin.com/in/rogermparent/) has already been a
+big part of building DVC and [CML](https://cml.dev/). He has been a primary
+developer of a Visual Studio code extension that interfaces with the DVC Python
+application to provide an interface with the Experiments feature that's coming
+out with DVC 2.0. We are so excited to have him joining us full time as Software
+Engineer.
+
+![Alt Text](https://media.giphy.com/media/vAvWgk3NCFXTa/giphy.gif)
+
+## Open Positions
+
+We are on the hunt for a
+[TypeScript Front-End Engineer](https://docs.google.com/document/d/1aT5HZYt4kAUxXqD4JNTe3jPDlVUwSmnEWDPR2QoKdvo/edit)
+to build SaaS and a VS Code UI for our popular machine learning tools: DVC and
+CML. The ML tools ecosystem is what JS space was 10 years ago. Come join us on
+this exciting project!
+
+Our search continues for a
+[Developer Advocate](https://weworkremotely.com/remote-jobs/iterative-developer-advocate)
+to support and inspire developers by creating new content like blogs, tutorials,
+and videos - plus lead outreach through meetups and conferences.
+
+Does this sound like you or someone you know? Be in touch!
+
+## Iterative.ai Featured on The New Stack
+
+[Susan Hall](https://thenewstack.io/author/susanhall/) of
+[The New Stack.io](https://thenewstack.io/) interviewed our very own CEO,
+[Dmitry Petrov](https://twitter.com/fullstackml?lang=en), discussing the needs
+of ML engineers and how Iterative.ai makes tools to enable version control and
+CI/CD for versioning data and ML models.
+
+> "ML engineers, they still need collaboration. They need GitHub for
+> collaboration, they need this CI?CD system to resolve [issues] between each
+> other, between the team and productions system." - Dmitry Petrov
+
+<external-link
+href="https://thenewstack.io/iterative-ai-git-based-machine-learning-tools-for-data-engineers/"
+title="Learning Tools for ML Engineers"
+description="Susan Hall"
+link="thenewstack.io"
+image="/uploads/images/2021-02-11/newstack_image.png"/>
+
+## Workshops and Talks
+
+### Developer Advocacy for Data Science
+
+So you saw the post further up. 👆🏽 Curious about developer advocacy or what to
+look for in a hire for this position?
+[Elle O'Brien](https://twitter.com/drelleobrien?lang=en) dove into this recently
```

Comment:
minor cleanup - can we remove `?lang=en` in a few links there? (it forces user interface changes I think on Twitter)